SampleScripts/PowerManagement-ManagementDomain.ps1

```diff
@@ -449,35 +447,35 @@ if ($PsBoundParameters.ContainsKey("shutdown") -or $PsBoundParameters.ContainsKe
                 $esxi_block["user"] = $esxDetails.username
                 $Pass = $esxDetails.password
                 if ($Pass) {
-                    $Pass_encrypted = $Pass | ConvertTo-SecureString -AsPlainText -Force | ConvertFrom-SecureString
+                    $PassEncrypted = $Pass | ConvertTo-SecureString -AsPlainText -Force | ConvertFrom-SecureString
                 } else {
-                    $Pass_encrypted = $null
+                    $PassEncrypted = $null
                 }
                 $esxi_block["password"] = $Pass_encrypted
                 $var["Hosts"] += $esxi_block
             }
 
             # Gather NSX Manager Cluster Details
             $nsxtCluster = Get-VCFNsxtCluster -id $workloadDomain.nsxtCluster.id
-            $nsxtMgrfqdn = $nsxtCluster.vipFqdn
+            $nsxtMgrFQDN = $nsxtCluster.vipFqdn
```

Comment:
What about `nsxManagerFqdn`?

Lower it flips to `nsx...`.

Reply:
Changed to nsxtManagerFQDN. "FQDN" looks better in upper case and all other variables around are "nsxt" - we may revisit this to transition to nsx, since this is the new official name of the product.